README.md

```diff
@@ -61,5 +68,10 @@ and `<custom.xyz>` refer to the options from the two configuration files, respec
 Note that the custom `jvmOpts` appear after the static `jvmOpts` and thus typically take precendence; the exact
 behaviour may depend on the Java distribution.
 
+`env` supports a very limited set of automatic expansions if you surround the expansion variable with `%%` as shown above,
+for `CUSTOM_PATH`. The following expansions are supported:
+
+* `%%.CWD%%`: The current working directory of the user which executed this process
```

Comment:
Canonical given what reference world? 

{{}} feels weird to me because {} is actually a reserved character in YAML:

http://www.yaml.org/spec/1.2/spec.html

% is too, but it's used to denote a directive line. 

I wish I could get rid of the `.CWD`, but unsure how with Go templates.